### ISSUE TITLE
[QUESTION] Losing predefined media queries

### ISSUE THREAD

**Issue by D33Funky** [artf/grapesjs#2184]
Hi. I have a predefined json of a template which contains a set of html elements and styles which are parsed into the set of components upon grapesjs load. The css contains some of the default media query rules which I'd like to keep for the template. Commands editor.getHTML() and editor.getCSS() return correct values when the page is loaded. I want to have the possibility to modify the html and css code separately and rebuild the components.
If I try to use the same html and css code to save the template, my predefined media queries are lost after the components are reset at the canvas:

In short, I'm doing the following:

```
let initialHtml = editor.getHtml(); // returns the correct html
let initialCss = editor.getCss(); // returns the correct css with the predefined set of media queries from JSON

editor.CssComposer.clear();

editor.setComponents(initialHtml);
editor.setStyle(initialCss);

let css = editor.getCss(); // at this point the initial media queries are missing even though they were provided in initialCss variable.
```

Can you please give a hint for the workaround?

**Comment by artf** [comment]
Provide a reproducible demo, please

**Comment by abozhinov** [comment]
I have the same problem. This is my issue -> https://github.com/artf/grapesjs/issues/2843
